visualizer/src/Controls/Tracking/Alerts/AlertGroup.js

```diff
@@ -0,0 +1,78 @@
+import { IconButton } from '@material-ui/core';
+import Collapse from '@material-ui/core/Collapse';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Alert from '@material-ui/lab/Alert';
+import clsx from 'clsx';
```

Comment:
What is `clsx`?

Reply:
It's a library for manipulating classNames: [https://www.npmjs.com/package/clsx](https://www.npmjs.com/package/clsx)

We're using it to add formatting when opening the alert boxes

It's installed as part of our dependencies, but I'm realizing that it's not explicitly in our package.json, so I'll add that in